lib/browser/remote/server.ts

```diff
@@ -234,29 +333,29 @@ const unwrapArgs = function (sender, frameId, contextId, args) {
         return callIntoRenderer
       }
       default:
-        throw new TypeError(`Unknown type: ${meta.type}`)
+        throw new TypeError(`Unknown type: ${(meta as any).type}`)
     }
   }
   return args.map(metaToValue)
 }
 
-const isRemoteModuleEnabledImpl = function (contents) {
-  const webPreferences = contents.getLastWebPreferences() || {}
+const isRemoteModuleEnabledImpl = function (contents: electron.WebContents) {
+  const webPreferences = (contents as any).getLastWebPreferences() || {}
```

Comment:
Why do you need to cast to `any` here?

Reply:
bc `getLastWebPreferences` isn't in the typings. i guess i could put it there? but i'm lazy :D